### PR TITLE
Add "WEMOS LoLin32" boards support.

### PR DESCRIPTION
**Pull request by wemos**
@projectgus 
@me-no-dev 
Add "WEMOS LoLin32" boards support.
Thank you.

### CODE REVIEW

File: variants/lolin32/pins_arduino.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
+
+#define LED_BUILTIN 2
+#define KEY_BUILTIN 0
+
+static const uint8_t SDA = 21;
+static const uint8_t SCL = 22;
+
+static const uint8_t SS    = 2;
```
Comment (me-no-dev) [inline variants/lolin32/pins_arduino.h]: shouldn't SS be GPIO 5?

Comment (wemos) [inline variants/lolin32/pins_arduino.h]: The ESP32's "General purpose SPI" can be remap to any GPIO

Comment (me-no-dev) [inline variants/lolin32/pins_arduino.h]: true, but it will not work just by defining it here ;)

Comment (wemos) [inline variants/lolin32/pins_arduino.h]: I agree with you, changed the SS to GPIO5.
Thank you.

**Pull request merged**